matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/timeline/DefaultTimeline.kt

```diff
@@ -81,8 +85,11 @@ internal class DefaultTimeline(
         private val timelineInput: TimelineInput,
         private val eventDecryptor: TimelineEventDecryptor,
         private val realmSessionProvider: RealmSessionProvider,
-        private val loadRoomMembersTask: LoadRoomMembersTask
-) : Timeline,
+        private val loadRoomMembersTask: LoadRoomMembersTask,
+        private val session: Session,
```

Comment:
If you have session you have crypto service. Maybe just use RoomGetter and crypto

Reply:
i am not sure it could work, the factory is not udpated?

Reply:
Done.